integration_test/third_party_apps_data/test_config.yaml

```diff
@@ -1,5 +1,13 @@
 per_application_overrides:
   apache:
+    # TODO: reenable distros incrementally.
+    platforms_to_skip:
+      - centos-7
+      - centos-8
+      - rocky-linux-8
+      - sles-12
+      - sles-15
+  cassandra:
     # Skip all platforms listed above except debian-10 for now.
     # New applications should use platforms_to_skip sparingly
     # if at all.
```

Comment:
Does this comment belong on the first use of `platforms_to_skip` above?

Reply:
sure, moved.